maintainers/maintainer-list.nix

```diff
@@ -13641,4 +13641,15 @@
       fingerprint = "3586 3350 BFEA C101 DB1A 4AF0 1F81 112D 62A9 ADCE";
     }];
   };
+  jcdickinson = {
```

Comment:
Please, add yourself to maintainer list as separate commit.

Reply:
thanks for the review, I have moved the change to a prior commit.